### PR TITLE
chore(CHANGELOG): update to v1.0.0 👿  ➡️ 

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

File: CHANGELOG.md
Comment (pinussilvestrus) [inline CHANGELOG.md:59]: Shall we keep the alpha versions or simply focus on the stable release here?

I'm asking because we recently did the same in bpmn-js: https://github.com/bpmn-io/bpmn-js/commit/d17e175c754cfa4baf4f1b69777a6465a767c49d

Comment (barmac) [reply CHANGELOG.md:59]: I could join the alphas but then it's a huge changelog update.

Comment (nikku) [reply CHANGELOG.md:73]: Proposal: Summarize most notable changes under the umbrella of _COmplete Rework_.

Comment (nikku) [reply CHANGELOG.md:73]: Also, link / show _all_ breaking changes.

Comment (pinussilvestrus) [reply CHANGELOG.md:73]: Already discussed this with @barmac and we will handle it similarly as you proposed 👍

**Comment by barmac** [comment]
@nikku @pinussilvestrus What do you think about how it looks now? Breaking changes and the biggest changes in the library are on top now and I refer to the alphas for more content.

**Comment by pinussilvestrus** [comment]
I like it pretty much 👍  Would wait for a second thought before approval ✅

File: CHANGELOG.md
Comment (nikku) [inline CHANGELOG.md:10]: I'd add a "spirit of this release on top", i.e. 

```
_A complete rewrite of the library, based on modern foundations._
```

Comment (barmac) [reply CHANGELOG.md:10]: As you wish: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/666/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR11

**Review by nikku — APPROVED** [review]

**Pull request merged**
